MANIFEST.in

```diff
@@ -1,5 +1,17 @@
-include *.rst *.txt
+include *.rst *.txt LICENSE
+exclude .editorconfig
+exclude .gitchangelog.rc
+exclude .hound.yml
+exclude .importanizerc
+exclude .pre-commit-config.yaml
+exclude Makefile
+exclude docker-compose.yml
+exclude pyproject.toml
+exclude pytest.ini
+exclude tox.ini
+recursive-include docs *
 recursive-include tests *
```

Comment:
same here. any reason to include tests? docs I usually find useful as you can see how to use the package if you only have access to the release package. tests not sure. thoughts?

Reply:
no reason, we can drop

Reply:
actually will include it in sdist, the sdist tarball should just be like `git ls-files`